conjur/logic/credentials_from_file.py

```diff
@@ -16,12 +16,15 @@
 # Internals
 from conjur.constants import DEFAULT_NETRC_FILE
 
+
+# TODO: Break this class to reader and writer classes and divide each one to generic class for read/write and  class for netrc that inherits it.
```

Comment:
Would it be better to file an issue that references this line instead of adding a `TODO`?

Reply:
Added this issue and removed TODO
https://github.com/cyberark/conjur-api-python3/issues/173